packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -213,17 +220,23 @@ async function runAxeTestsForBlockEditor() {
  * Simulate slow network or throttled CPU if provided via environment variables.
  */
 async function simulateAdverseConditions() {
-	if ( ! DOWNLOAD_THROUGHPUT && ! THROTTLE_CPU ) {
+	if ( ! SLOW_NETWORK && ! OFFLINE && ! THROTTLE_CPU ) {
 		return;
 	}
 
 	const client = await page.target().createCDPSession();
 
-	if ( DOWNLOAD_THROUGHPUT ) {
+	if ( SLOW_NETWORK || OFFLINE ) {
 		// See: https://chromedevtools.github.io/devtools-protocol/tot/Network#method-emulateNetworkConditions
 		await client.send( 'Network.emulateNetworkConditions', {
-			// Simulated download speed (bytes/s)
-			downloadThroughput: Number( DOWNLOAD_THROUGHPUT ),
+			// Network connectivity is absent
+			offline: Boolean( OFFLINE || false ),
+			// Download speed (bytes/s)
+			downloadThroughput: ( ( 1.6 * 1024 * 1024 ) / 8 ) * 0.9,
+			// Upload speed (bytes/s)
+			uploadThroughput: ( ( 750 * 1024 ) / 8 ) * 0.9,
+			// Latency (ms)
+			latency: 150 * 3.75,
```

Comment:
These numbers (particularly `150` and `0.9`) are a bit "magic" so I think let's link to https://github.com/ChromeDevTools/devtools-frontend/blob/80c102878fd97a7a696572054007d40560dcdd21/front_end/sdk/NetworkManager.js#L252-L274 in an inline comment.